wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -229,6 +229,15 @@ class Command {
    */
   virtual ConditionalCommand Unless(std::function<bool()> condition) &&;
 
+  /**
+   * Decorates this command to run or stop when disabled.
+   *
+   * @param doesRunWhenDisabled true to run when disabled.
+   * @return the decorated command
+   */
+  virtual std::unique_ptr<Command> WithRunsWhenDisabled(
+      bool doesRunWhenDisabled) &&;
```

Comment:
Is this actually usable right now?  It seems like this won't work for inline usage until unique_ptr support is fixed?

Reply:
Afaik it's fully usable, with the sole gotcha that it would need to be dereferenced.
Nothing about `unique_ptr` is broken atm; those changes are to increase support for it and reduce the need to dereference the `unique_ptr` and is primarily a docs / recommended usage change.